nomad/csi_endpoint.go

```diff
@@ -459,7 +459,7 @@ func (v *CSIPlugin) Get(args *structs.CSIPluginGetRequest, reply *structs.CSIPlu
 			}
 
 			if plug != nil {
-				plug, err = state.CSIPluginDenormalize(ws, plug)
+				plug, err = state.CSIPluginDenormalize(ws, plug.Copy())
```

Comment:
ugh

Reply:
This feels like the sort of thing that's going to be easy to mess up, but I don't have a better solution at the moment.